client/blocks/reader-post-card/featured-video.jsx

```diff
@@ -15,7 +15,7 @@ import FeaturedImage from './featured-image';
 class FeaturedVideo extends React.Component {
 
 	static propTypes = {
-		thumbnailUrl: React.PropTypes.string,
+		thumbnailUrlPromise: React.PropTypes.object,
```

Comment:
> passing in a promise as a prop seems like a reasonable compromise actually, and I've been trying to reconsider promise-returning actions lately though I am not convinced of their value. something like this feels like an exception.

Thanks for the sanity check here @dmsnell. I'd be fine with deferring to whatever solution the reader folks come up with, since they'll be the main maintainers. As you noted this does seem like an exception case.